dashboard-backend/go.mod

```diff
@@ -0,0 +1,57 @@
+module github.com/litmuschaos/e2e-dashboard-backend
+
+go 1.17
+
+require (
+	github.com/gin-contrib/cors v1.3.1
+	github.com/gin-gonic/gin v1.7.4
+	github.com/joho/godotenv v1.4.0
+	github.com/litmuschaos/litmus-go v0.0.0-20210705063441-babf0c4aa57d
+)
+
+require (
+	github.com/gin-contrib/sse v0.1.0 // indirect
+	github.com/go-playground/locales v0.14.0 // indirect
+	github.com/go-playground/universal-translator v0.18.0 // indirect
+	github.com/go-playground/validator/v10 v10.9.0 // indirect
+	github.com/golang/protobuf v1.5.2 // indirect
+	github.com/google/go-cmp v0.5.6 // indirect
+	github.com/json-iterator/go v1.1.12 // indirect
+	github.com/leodido/go-urn v1.2.1 // indirect
+	github.com/mattn/go-isatty v0.0.14 // indirect
+	github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd // indirect
+	github.com/modern-go/reflect2 v1.0.2 // indirect
+	github.com/sirupsen/logrus v1.7.0 // indirect
+	github.com/ugorji/go/codec v1.2.6 // indirect
+	golang.org/x/crypto v0.0.0-20210921155107-089bfa567519 // indirect
+	golang.org/x/sys v0.0.0-20211031064116-611d5d643895 // indirect
+	golang.org/x/text v0.3.7 // indirect
+	google.golang.org/protobuf v1.27.1 // indirect
+	gopkg.in/yaml.v2 v2.4.0 // indirect
+)
+
+replace (
```

Comment:
@Aman-Codes We don't require these, please use `go mod tidy` before committing the changes. cc: @uditgaurav

Reply:
you can minimise and run `go mod tidy`

Reply:
On removing any `replace` I get the following errors
![image](https://user-images.githubusercontent.com/54680709/142982274-51d5bb3e-57b6-4a45-aa9a-566f409d0a20.png)
cc @Jonsy13 @uditgaurav

Reply:
running `go mod tidy` didn't have any change